### PR TITLE
feat(shell): deploy lifebankcoin contract in lacchain

### LINKED ISSUE #534: Deploy Lifebank to LACChain
We need a permanent network for production purposes. Currently we have a development node in our network. 

LACChain offers a different topology. It is a permissioned blockchain and legally supervised. 

[Click here](https://docs.google.com/document/d/1-wXfuDPfVe1dUbWaQ6FApqZpYWzyhxq0jD2xkn67drw/edit?usp=sharing) to see the documentation process.

Own: @leisterfrancisco 
Sup: @steph222

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #534 

### What does this PR do?
Deploy lifebank to lacchain

### How should this be tested?
- Make run

### CODE REVIEW

File: contracts/scripts/lifebank.sh [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+}
+assign_resources() {
+    cleos -u $EOS_API_URL get account lbacccreator 
+    cleos -u $EOS_API_URL system buyram eosio lbacccreator "10000 EOS"
+    cleos -u $EOS_API_URL get account lbacccreator 
+    cleos -u $EOS_API_URL transfer eosio lbacccreator "10000000 EOS" "Lifebank initial funding"
+}
+deploy_lifebank_contracts() {
+    echo 'Deploy Consent Contract'
+    cleos -u $EOS_API_URL set contract consent2life ../consent2life/ -p consent2life@active -j -s -d
```
Comment (xavier506) [inline contracts/scripts/lifebank.sh:59]: we need to add the writer run action  and sign the transaction 

`-s` = skip signatures

**Review by Edgar097 — APPROVED** [review]

**Review by steph222 — APPROVED** [review]
Good work

File: contracts/lifebankcoin/lifebankcoin.abi [hunk 2]
```diff
@@ -126,27 +158,32 @@
         {
             "name": "clear",
             "type": "clear",
-            "ricardian_contract": "---\nspec-version: 0.1.0\ntitle: Clear table\nsummary: The intent of the `{{ clear }}` action is to delete all table records and is used for development purposes on test-nets and must not be published to mainnet.\nicon: https://lifebank.io/icons/admin.png"
+            "ricardian_contract": ""
```
Comment (xavier506) [inline contracts/lifebankcoin/lifebankcoin.abi:161]: @leisterfrancisco  seems you are deleting all the ricardian clauses , please recompile contracts with clauses

File: contracts/lifebanktest/src/lifebanktest.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#include <lifebanktest.hpp>
+
+ACTION lifebanktest::hi(name from, string message, string message2) {
```
Comment (xavier506) [inline contracts/lifebanktest/src/lifebanktest.cpp:3]: @leisterfrancisco  what are you using this action for?

**Review by xavier506 — COMMENTED** [review]
@leisterfrancisco  awesome job! I really like 👍 the shell scripts 💯 

please see my comments

File: contracts/lifebanktest/src/lifebanktest.cpp (hunk 3 above)
Comment (leisterfrancisco) [reply contracts/lifebanktest/src/lifebanktest.cpp:3]: This shouldn't be uploaded

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**